### PR TITLE
Adding remote write configs for Mongodb Metrics 

### PR DESCRIPTION
**Pull request by anphn-mtt**
###### Description

Added Remote/Write Configs for Mongodb metrics:
List of Metrics are on following page:
https://github.com/influxdata/telegraf/tree/master/plugins/inputs/mongodb

Metrics follow following formart.

```bash 

        ## mongodb_active_reads
        ## mongodb_active_writes
        ## mongodb_commands_per_sec
        ## mongodb_connections_current
        ## mongodb_db_stats_storage_size
        ## mongodb_deletes_per_sec
        ## mongodb_document_*
        ## mongodb_flushes_per_sec
        ## mongodb_getmores_per_sec
        ## mongodb_inserts_per_sec
        ## mongodb_net_*_bytes_count
        ## mongodb_open_connections
        ## mongodb_page_faults
        ## mongodb_percent_cache_dirty
        ## mongodb_percent_cache_used
        ## mongodb_queries_per_sec
        ## mongodb_queued_reads
        ## mongodb_queued_writes
        ## mongodb_repl_queries
        ## mongodb_repl_commands_per_sec
        ## mongodb_repl_deletes_per_sec
        ## mongodb_repl_getmores_per_sec
        ## mongodb_repl_inserts_per_sec
        ## mongodb_repl_oplog_window_sec
        ## mongodb_repl_queries_per_sec
        ## mongodb_repl_updates_per_sec
        ## mongodb_resident_megabytes
        ## mongodb_updates_per_sec
        ## mongodb_uptime_ns
        ## mongodb_vsize_megabytes
        ## mongodb_wtcache_bytes_read_into
 
 ```

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -2338,6 +2338,50 @@ kube-prometheus-stack:
...
+        ## mongodb_resident_megabytes
+        ## mongodb_updates_per_sec
+        ## mongodb_uptime_ns
+        ## mongodb_vsize_megabytes
+        ## mongodb_wtcache_bytes_read_into
+        - url: http://$(FLUENTD_METRICS_SVC).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.applications.mongodb
+          remoteTimeout: 5s
+          writeRelabelConfigs:
+            - action: keep
+              regex: (?:mongodb_(active_(reads|writes)|commands_.*|connections_current|db_stats_storage_size|deletes_.*|document_.*|flushes_.*|getmores_.*|inserts_.*|net_.*|open_connections|page_faults|percent_cache_(dirty|used)|queries_.*|queued_(reads|writes)|repl_.*|resident_megabytes|updates_per_sec|uptime_ns|vsize_megabytes|wtcache_bytes_read_into))
```
Comment (pmalek-sumo) [inline deploy/helm/sumologic/values.yaml:2382]: Ditto as elsewhere:

https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1748#discussion_r708037611

Thanks!

File: deploy/helm/sumologic/values.yaml [hunk 2]
```diff
@@ -2467,6 +2467,47 @@ kube-prometheus-stack:
...
+        ## mongodb_resident_megabytes
+        ## mongodb_updates_per_sec
+        ## mongodb_uptime_ns
+        ## mongodb_vsize_megabytes
+        ## mongodb_wtcache_bytes_read_into
+        - url: http://$(FLUENTD_METRICS_SVC).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.applications.mongodb
+          remoteTimeout: 5s
+          writeRelabelConfigs:
+            - action: keep
+              regex: (?:mongodb_(active_(reads|writes)|commands_per_sec|connections_current|db_stats_storage_size|deletes_per_sec|document_.*|flushes_per_sec|getmores_per_sec|inserts_per_sec|net_.*_bytes_count|open_connections|page_faults|percent_cache_(dirty|used)|queries_per_sec|queued_(reads|writes)|repl_((commands|deletes|getmores|inserts|oplog|queries|updates)_per_sec)|queries)|resident_megabytes|updates_per_sec|uptime_ns|vsize_megabytes|wtcache_bytes_read_into))
```
Comment (pmalek-sumo) [inline deploy/helm/sumologic/values.yaml:2508]:
```suggestion
              regex: (?:mongodb_(active_(reads|writes)|commands_per_sec|connections_current|db_stats_storage_size|deletes_per_sec|document_.*|flushes_per_sec|getmores_per_sec|inserts_per_sec|net_.*_bytes_count|open_connections|page_faults|percent_cache_(dirty|used)|queries_per_sec|queued_(reads|writes)|repl_((commands|deletes|getmores|inserts|oplog|queries|updates)_per_sec|queries)|resident_megabytes|updates_per_sec|uptime_ns|vsize_megabytes|wtcache_bytes_read_into))
```

Without this change this is an invalid regex:

File: deploy/helm/sumologic/values.yaml [hunk 3]
```diff
@@ -2467,6 +2467,47 @@ kube-prometheus-stack:
...
+        ## mongodb_queued_writes
+        ## mongodb_repl_commands_per_sec
+        ## mongodb_repl_deletes_per_sec
+        ## mongodb_repl_getmores_per_sec
+        ## mongodb_repl_inserts_per_sec
+        ## mongodb_repl_oplog_window_sec
+        ## mongodb_repl_queries_per_sec
+        ## mongodb_repl_updates_per_sec
+        ## mongodb_resident_megabytes
+        ## mongodb_updates_per_sec
```
Comment (pmalek-sumo) [inline deploy/helm/sumologic/values.yaml:2500]: Below added regex doesn't match this (checked with https://regex101.com/)

File: deploy/helm/sumologic/values.yaml [hunk 4]
```diff
@@ -2467,6 +2467,47 @@ kube-prometheus-stack:
...
+        ## mongodb_percent_cache_dirty
+        ## mongodb_percent_cache_used
+        ## mongodb_queries_per_sec
+        ## mongodb_queued_reads
+        ## mongodb_queued_writes
+        ## mongodb_repl_commands_per_sec
+        ## mongodb_repl_deletes_per_sec
+        ## mongodb_repl_getmores_per_sec
+        ## mongodb_repl_inserts_per_sec
+        ## mongodb_repl_oplog_window_sec
```
Comment (pmalek-sumo) [inline deploy/helm/sumologic/values.yaml:2496]: Below added regex doesn't match this (checked with https://regex101.com/)

Comment (anphn-mtt) [reply deploy/helm/sumologic/values.yaml:2497]: i updated and checked with https://regex101.com/).

[
Screen Shot 2021-09-22 at 09 25 39
](url)

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**